week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -43,19 +43,69 @@ Outlook not so good.
 Very doubtful.
 */
 
+let array = [
+  // Very Positive:
```

Comment:
Very good job laying out your array this way! It's very easy for me to read this as a result :)

Reply:
Thanks!